sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
 ## 1.0.0-preview.3 (Unreleased)
-
+- Renamed type `SentimentScorePerLabel` to `SentimentConfidenceScorePerLabel`.
+- In both `DocumentSentiment` and `SentenceSentiment` property `SentimentScores` has been renamed to `ConfidenceScores`.
+- In `LinkedEntity`, property `Id` has been renamed to `DataSourceEntityId`.
+- Added `DetectedLanguageInput.None` for user convenience, instead of passing empty string to `CountryHint`.
```

Comment:
Is the class name `DetectedLanguageInput` or `DetectLanguageInput` ?

Reply:
You are right. Thanks! will change